galaxy.yml

```diff
@@ -7,7 +7,7 @@ authors:
   - "Michał Nasiadka"
   - "Mark Goddard"
 dependencies:
-  "pulp.squeezer": "*"
+  "pulp.squeezer": "0.0.11"
```

Comment:
Like I said in the other review, this collection is more agnostic to the squeezer version.